framework/src/node/node.ts

```diff
@@ -626,8 +626,7 @@ export class Node {
 			this._logger.error({ err: err as Error }, 'Failed to load delegates for forging');
 		}
 		this._forgingJob = new jobHandlers.Scheduler(async () => this._forgingTask(), forgeInterval);
-		// eslint-disable-next-line @typescript-eslint/no-floating-promises
-		this._forgingJob.start();
+		await this._forgingJob.start();
```

Comment:
This await might have been left out to avoid blocking here; maybe check with original author. Rest of PR looks good.